src/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ option(BUILD_TESTS          "Build tests"                  ON)
 option(VARIORUM_WITH_INTEL  "Support Intel architectures"  ON)
 option(VARIORUM_WITH_AMD    "Support AMD architectures"    ON)
 option(VARIORUM_WITH_IBM    "Support IBM architectures"    ON)
```

Comment:
Let's disable `VARIORUM_WITH_AMD` and `VARIORUM_WITH_IBM` as well. There's no external dependencies like there is for Nvidia platforms, but many of the systems we see this deployed on are Intel-based.

Reply:
Done!